src/fklearn/validation/validator.py

```diff
@@ -63,6 +67,8 @@ def validator_iteration(data: pd.DataFrame,
     warnings.warn(empty_set_warn) if train_data.shape[0] == 0 else None  # type: ignore
 
     predict_fn, train_out, train_log = train_fn(train_data)
+    if return_train_score:
+        train_log = assoc(train_log, "eval_results", eval_fn(train_out))
```

Comment:
Just a quick question, why do you choose `assoc` instead of this?
```suggestion
        train_log['eval_results'] = eval_fn(train_out)
```

Reply:
Hi Eric! Thanks for your comments! The `assoc` function is used multiple times in this file, everytime a new key-value pair is added to a dictionary, so I used it to be consistent. I guess it's more functional style than assigning the value to the new key because instead of mutating the already existing dictionary the `assoc` function returns a new dictionary with the new key-value pair added to it.

Reply:
That's a great reason, consistency is always important, thanks for your answer Diego!